### PR TITLE
Styling components admin

### PR DESCRIPTION
**Pull request by BukovskyiDima**
change some styles components tests and main, add variables
fixed images

### CODE REVIEW

**Comment by BukovskyiDima** [comment]
@iliatcymbal PTAL

File: client/components/admin/admin.html [hunk 1]
```diff
@@ -21,7 +21,7 @@
             <p>
             <h1>Имя теста</h1>
             <input type="text"
-                   class="form-control"
+                   class="form-control form-control_style"
```
Comment (iliatcymbal) [inline client/components/admin/admin.html]: According to BEM model you shuld use double underscore __: `form-control__style`. Also it's better to use more informative name, i.e. `form-control__field`

File: client/components/admin/admin.html [hunk 2]
```diff
@@ -30,33 +30,33 @@
...
+                    <a class="btn btn-default btn_style"
                        ng-if="admin.editableField > -1"
                        ng-click="admin.saveQuestion()">
                         Сохранить изменения
                     </a>
                 </div>
 
                 <div class="st-admin__fields">
-                    <select class="form-control"
+                    <select class="form-control form-control_style"
```
Comment (iliatcymbal) [inline client/components/admin/admin.html]: Use BEM rules to set classes and more informative class name. "style" it's not the proper name for elements

File: client/components/admin/admin.html [hunk 3]
```diff
@@ -69,21 +69,21 @@
                                    class="st-admin__flag"
                                    ng-class="{'form-control': admin.typeAnswer.type == 'text'}">
                             <input ng-if="admin.typeAnswer.type != 'text'"
-                                   class="form-control st-admin__variant"
+                                   class="form-control form-control_style form-control_style-variant st-admin__variant"
```
Comment (iliatcymbal) [inline client/components/admin/admin.html]: Use BEM rules to set classes and more informative class name. "style" it's not the proper name for elements
Also for elements BEM require to use double underscore __
For what reason you've set "form-control_style-variant" class, isn't enough "st-admin__variant"?

File: client/components/admin/admin.html [hunk 4]
```diff
@@ -69,21 +69,21 @@
...
                                    type="text"
                                    ng-model="admin.form.variants[$index].value">
 
                             <div class="btn-group">
                                 <button type="button"
-                                        class="btn btn-success"
+                                        class="btn btn-success btn_style btn_style-variant btn_style-variant-add"
```
Comment (iliatcymbal) [inline client/components/admin/admin.html]: "-add" - it's modificator, according BEM model you should use double dash --
`btn__style-variant--add`

File: client/components/admin/admin.html [hunk 5]
```diff
@@ -69,21 +69,21 @@
...
                                         ng-click="admin.addVariant()">Add</button>
 
                                 <button type="button"
-                                        class="btn btn-danger"
+                                        class="btn btn-danger btn_style btn_style-variant btn_style-variant-remove"
```
Comment (iliatcymbal) [inline client/components/admin/admin.html]: "-remove" - it's modificator, according BEM model you should use double dash --
'btn__style-variant--remove`

File: client/components/admin/admin.html [hunk 6]
```diff
@@ -69,21 +69,21 @@
...
                                         ng-click="admin.removeVariant($index)">Remove</button>
 
                                 <button type="button"
-                                        class="btn btn-info"
+                                        class="btn btn-info btn_style btn_style-variant btn_style-variant-correct"
```
Comment (iliatcymbal) [inline client/components/admin/admin.html]: "-correct" - it's modificator, according BEM model you should use double dash --
'btn__style-variant--correct`

File: client/components/admin/admin.html [hunk 7]
```diff
@@ -115,17 +115,17 @@
                 </li>
             </ul>
 
-            <button class="btn btn-default"
+            <button class="btn btn-default btn_style btn_style-control btn_style-control-edit"
```
Comment (iliatcymbal) [inline client/components/admin/admin.html]: name classes according to BEM

File: client/components/admin/admin.html [hunk 8]
```diff
@@ -115,17 +115,17 @@
...
                     type="button"
                     ng-click="admin.editQuestion($index)">Edit</button>
-            <button class="btn btn-default"
+            <button class="btn btn-default btn_style btn_style-control btn_style-control-delite"
```
Comment (iliatcymbal) [inline client/components/admin/admin.html]: name classes according to BEM

File: client/components/admin/admin.html [hunk 9]
```diff
@@ -115,17 +115,17 @@
...
                     type="button"
                     ng-click="admin.deleteQuestion($index)">Delete</button>
 
         </li>
     </ul>
 
-    <button class="btn btn-default"
+    <button class="btn btn-default btn_style btn_style-save"
```
Comment (iliatcymbal) [inline client/components/admin/admin.html]: name classes according to BEM

File: client/components/admin/admin.scss [hunk 10]
```diff
@@ -19,21 +33,55 @@
...
     .form-control{
       margin: 0 10px;
     }
 
     .btn-group{
       min-width: 200px;
+
+      .btn_style-variant {
+        width: 99px;
+        margin: 0px 0 0 5px !important;
```
Comment (iliatcymbal) [inline client/components/admin/admin.scss]: do not use 'px' near zero value

File: client/components/admin/admin.scss [hunk 11]
```diff
@@ -19,21 +33,55 @@
...
+        background-color: $list-test-color !important;
+      }
+
+      .btn_style-variant-correct {
+        background-color: #5faeea;
+      }
+
+      .btn_style-variant-correct:focus,
+      .btn_style-variant-correct:hover {
+        background-color: #5faeea !important;
```
Comment (iliatcymbal) [inline client/components/admin/admin.scss]: try do not use `!important` directive, it's monkey code

File: client/components/admin/admin.scss [hunk 12]
```diff
@@ -72,10 +128,73 @@
...
   }
 
   &__notes{
     white-space: pre-wrap;
     font-family: monospace;
   }
 }
+
+.form-control_style, .btn_style{
+  border-radius: $input-border-radius !important;
```
Comment (iliatcymbal) [inline client/components/admin/admin.scss]: `!important` - monkey code

File: client/components/admin/admin.scss [hunk 13]
```diff
@@ -72,10 +128,73 @@
...
+
+.form-control_style-variant {
+  font-size: $input-font-size-placeholder;
+  font-family: $input-font-family-placeholder;
+  color: #373737;
+}
+
+.btn:hover,
+.btn:focus{
+  outline: none !important;
```
Comment (iliatcymbal) [inline client/components/admin/admin.scss]: don't use `important`  everywhere!

File: logs.json [hunk 14]
```diff
@@ -145,3 +145,135 @@
 {"level":"info","message":"\u001b[90mGET /rests/tests\u001b[39m\u001b[32m 200 \u001b[39m\u001b[90m2ms\u001b[39m","timestamp":"2016-10-09T18:46:34.646Z"}
```
Comment (iliatcymbal) [inline logs.json]: we don't need in repo this file

File: client/components/admin/admin.html [hunk 15]
```diff
@@ -30,33 +30,33 @@
...
                            ng-model="admin.form.question">
 
-                    <textarea class="form-control"
+                    <textarea class="form-control form-control_style"
                               cols="30"
                               rows="5"
                               ng-model="admin.form.notes"></textarea>
 
-                    <button class="btn btn-default"
+                    <button class="btn btn-default btn_style btn_style-control"
```
Comment (BukovskyiDima) [inline client/components/admin/admin.html]: can i change the name on 'btn__button btn__button-control' or better 'btn__style btn__style-control'?

**Comment by BukovskyiDima** [comment]
@iliatcymbal PTAL and can you write mу or can i call you? i have some question.

**Comment by BukovskyiDima** [comment]
@iliatcymbal  PTAL

**Comment by BukovskyiDima** [comment]
@iliatcymbal PTAL you must acctept

**Comment by BukovskyiDima** [comment]
@iliatcymbal can i merge?

**Pull request merged**